bust-a-move-framework/src/test/java/bustamove/player/PlayerTest.java

```diff
@@ -0,0 +1,82 @@
+package bustamove.player;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.Before;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Score Tester
+ * @author Maurice Willemsen
+ * @version 17.10.2016
+ */
+@RunWith(Parameterized.class)
+public class PlayerTest {
+
+	private Player player;
+	private TestObserver tester;
+	
+    @Parameterized.Parameters
+    public static Object[] input() {
+        return new Object[][]{{new Player("TestPlayer",0,1)}};
+    }
+
+    public PlayerTest(Object expected) {
+    	player = (Player) expected;
+    }
+
+    @Before
+    public void setUp() {
+    	tester = new TestObserver();
+    }    
+    
+    @Test
+    public void setName() {
+    	player.setName("NewTestPlayer");
+    	assertThat("Score should be added", player.getName(), equalTo("NewTestPlayer"));		

```

Comment:
I generally leave out the message myself because if a test fails, it should be immediately clear where/what failed.